### PR TITLE
fix(dedicated): private cloud mailing list

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-5531
| License          | BSD 3-Clause

## Description

Mailing list is not applicable for the US region. Hide mailing list section for the US region.

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/tiles/service-management/service-management.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <oui-tile-definition
         data-term="{{:: 'ovhManagerPccDashboardServiceManagement_mailingList_term' | translate }}"
         data-description="{{:: $ctrl.currentUser.email }}"
+        data-ng-if="::$ctrl.currentUser.ovhSubsidiary !== 'US'"
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/tiles/service-management/service-management.html:8]: Shouldn't be based on the region over the subsidiary?

Comment (Jisay) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/tiles/service-management/service-management.html:8]: Maybe it's also the good time to use feature flipping?

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/tiles/service-management/service-management.html]: @marie-j has started configured it for the dedicated application. 

see: https://github.com/ovh/manager/pull/3466/files?file-filters%5B%5D=.html&file-filters%5B%5D=.json#diff-ea0f3ce3d53e95c8861f210b691dac0dR327-R329

**thought:** maybe we can rebase this Pull Request on `fix/hpc-us`.

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#14595.0 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by radireddy** [comment]
created one more PR for this change https://github.com/ovh/manager/pull/3478. This is no longer required

**Pull request closed without merging**